Kconfig

```diff
@@ -839,22 +839,19 @@ menu "LVGL configuration"
 
     menu "3rd Party Libraries"
         config LV_USE_FS_STDIO
-            int "File system on top of stdio API"
-            default 0
+            bool "File system on top of stdio API"
```

Comment:
Here you need to set an ASCII character (i.e. a driver letter) on which LVGL will reach this file system.
Finally, a file path will look like `"S/my/folder/file.txt"`.

Reply:
Right. Is this the same for the other two too? Can we just add a sub-option for the filesystem letter?

Reply:
> Is this the same for the other two too

Yes, same for the others too. 

> Can we just add a sub-option for the filesystem letter?

I'm ok with that. User can choose from ASCII upper case letters and `\0` to disable.

Reply:
or default set to "" for path starting with separator "/"

Reply:
@rzr It's only a character, not a string :slightly_frowning_face: 
E.g. `'A'`

Reply:
yes but my point that string would be more generic like "A:" (or "df0:" for amiga os ;) )

Reply:
Yes, it'd be really better but would be an API break. We can do it only in v9.